### ISSUE TITLE
Auto Scrolling Sheets

### ISSUE THREAD

**Issue by nevetS-718** [SheetAble/SheetAble#41]
When displaying a sheet of music with multiple pages, have a feature that can scroll through the music with a BPM control scroll speed. Sheets would scroll continuously top to bottom.

![Scroll](https://user-images.githubusercontent.com/70534545/176176288-177d1cd9-836c-4ddb-b59b-223da68de94e.png)

**Comment by Drumhavoc** [comment]
Even if it's not automatic scroll based on tempo, continuous scroll would be great for smooth transition between pages. Maybe put it in the options for continuous scroll or page turn so the user can decide. Same should be for landscape mode.

**Comment by vallezw** [comment]
Yes sounds good when looking at the tablet client though, I will probably add all the settings in the top navbar. Like there will be when adding sketchig

**Comment by ACCakut** [comment]
I think, the standard way of scrolling/turning pages is a finger tip or button controller. Scroll by BPM has issues, mainly its static speed (so it "miss"scrolls is something is repeated or you interrupt).

A full screen view in the web UI would be great and make the app less important as well es easier to implement, I think.
